Gordon360/Models/ACCOUNT.cs

```diff
@@ -31,5 +31,6 @@ public partial class ACCOUNT
         public int is_police { get; set; }
         public Nullable<int> Chapel_Required { get; set; }
         public string Mail_Location { get; set; }
+        public Nullable<int> Chapel_Attended { get; set; }
```

Comment:
Just out of curiosity, was this added when you merged your branch with the develop branch?